docker/README.md

```diff
@@ -0,0 +1,83 @@
+### About this Dockerfile
+... to be added
```

Comment:
How about:

```
# Dockerfile for EMSDK 

This Dockerfile builds a self-contained version of emsdk that enables emscripten to be used without any
other installation on the host system.

It is pusblished at https://hub.docker.com/u/emscripten/emsdk.

```

Reply:
Is https://hub.docker.com/u/emscripten the official profile of emscripten-core?

Reply:
As of just recently yes.  I think we just created it yesterday for the emscripten-ci image

Reply:
Not sure if we should call this image `emscripten/emsdk` or `emscripten/emscripten`?  I think maybe the former which this is where the dockerfile has ended up and the image does contain the emsdk?

Reply:
Surprisingly hard question, `emscripten` will be more popular as the name has better recognition. 
If we'd go with a name `emsdk` it would be technically correct to have just `emscripten-core/emsdk` installed without specific emscripten sdk being pulled, where this image is profiled to deliver a concrete sdk version. 
My vote is for `emscripten`, but again. It's your call.

Reply:
OK, lets go with emscripten/emscripten to keep it simple.